lib/render/editor/editor.html

```diff
@@ -570,17 +622,20 @@
         return isWaiting ? "visible" : "hidden";
       },
 
-      _resetBoard: function () {
-        const initialState = this.initialState;
-        const initialBoard = this._getTargetBoard();
-        initialBoard.update(initialState.table, initialState.header);
-        initialBoard.boom = false;
+      _resetBoards: function () {
+        const boards = this._getInitialBoards();
+        this.initialState.forEach(({table, header, boom}, index) => {
+          let board = boards[index];
+          board.update(table, header);
+          board.boom = boom;
```

Comment:
When resetting, since we are clearing all the blockly errors, we should set `boom` to false, right?

Reply:
Boom's value comes from the initialState, so it would be set to false unless the initialBoard is a boom (which shouldn't happen)